Documentation/benchmarks/etcd-3-watch-memory-benchmark.md

```diff
@@ -0,0 +1,72 @@
+# Watch Memory Usage Benchmark
+
+*NOTE*: The watch features are still being actively developed and we expect the memory usage to change over time. However we do expect that memory usage should not be significantly increased from the figures below.
```

Comment:
It's more "we currently eat 3 meals a day, but we're still adjusting our
diet, and the exact amount consumed will fluctuate during this process;
however, once we've settled on a final diet, we don't expect food intake to
be significantly higher than these preliminary figures"

On Fri, Nov 13, 2015, 15:44 Josh Wood <notifications@github.com> wrote:

> In Documentation/benchmarks/etcd-3-watch-memory-benchmark.md
> <https://github.com/coreos/etcd/pull/3862#discussion_r44847212>:
>
> > @@ -0,0 +1,72 @@
> > +# Watch Memory Usage Benchmark
> > +
> > +*NOTE*: The watch features are still being actively developed and we expect the memory usage to change over time. However we do expect that memory usage should not be significantly increased from the figures below.
>
> The honest truth is, I don't know that the "Note" here serves any real
> purpose. It says almost nothing. If we mean "We currently eat memory for 5
> meals a day, but will go on a diet in future development." then we should
> say that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/3862/files#r44847212>.
>